ilastik/applets/tracking/conservation/conservationTrackingGui.py

```diff
@@ -349,16 +349,6 @@ def _handle_failure( exc, exc_info ):
         req.notify_finished( _handle_finished )
         req.submit()
 
-    def menus(self):
-        menus = super( ConservationTrackingGui, self ).menus()
-        
-        m = QtGui.QMenu("&Export", self.volumeEditorWidget)
-        m.addAction("Export Tracking Information").triggered.connect(self.show_export_dialog)
```

Comment:
I think we can safely drop this menu item on all versions of ilastik, so I agree, this is the fix to the bug #1451